test/e2e/framework/psp/psp.go

```diff
@@ -85,13 +84,13 @@ func IsPodSecurityPolicyEnabled(kubeClient clientset.Interface) bool {
 	isPSPEnabledOnce.Do(func() {
 		psps, err := kubeClient.PolicyV1beta1().PodSecurityPolicies().List(metav1.ListOptions{})
 		if err != nil {
-			e2elog.Logf("Error listing PodSecurityPolicies; assuming PodSecurityPolicy is disabled: %v", err)
+			Logf("Error listing PodSecurityPolicies; assuming PodSecurityPolicy is disabled: %v", err)
```

Comment:
good catch from CI:
```
/b/f/w/test/e2e/framework/psp/psp.go:87:4: undefined: Logf
/b/f/w/test/e2e/framework/psp/psp.go:90:4: undefined: Logf
/b/f/w/test/e2e/framework/psp/psp.go:93:4: undefined: Logf
/b/f/w/test/e2e/framework/psp/psp.go:163:3: undefined: Logf
```

I think you'll have to import the framework
```go
import (
    ...
    "k8s.io/kubernetes/test/e2e/framework"
    ...
)

framework.Logf(...)
```

Reply:
No, this file just in `framework` package, `Logf` should have been used.